--- old/btusb.c	2026-08-05 05:55:05.024589302 +0000
+++ new/btusb.c	2026-08-05 05:55:05.027821316 +0000
@@ -797,6 +797,7 @@
 	{ USB_DEVICE(0x13d3, 0x3394), .driver_info = BTUSB_REALTEK },
 
 	/* Additional Realtek 8723BE Bluetooth devices */
+	{ USB_DEVICE(0x0930, 0x0222), .driver_info = BTUSB_REALTEK },
 	{ USB_DEVICE(0x0489, 0xe085), .driver_info = BTUSB_REALTEK },
 	{ USB_DEVICE(0x0489, 0xe08b), .driver_info = BTUSB_REALTEK },
 	{ USB_DEVICE(0x04f2, 0xb49f), .driver_info = BTUSB_REALTEK },
